symfony/plugins/orangehrmAdminPlugin/web/js/workShiftSuccess.js

```diff
@@ -217,7 +217,7 @@ function getWorkShiftEmpInfo(url){
             // Use Jquery to get select list element
             var dropdownList = $("#workShift_assignedEmp")[0];
 
-            if ($.browser.msie) {
+            if (/Edge\/|Trident\/|MSIE /.test(window.navigator.userAgent)) {
```

Comment:
may be a new global function identified is_msie()

Reply:
global function has been added to archive.js which is included in layout.